modules/calibration/data/dev_kit_advanced_3lidar/planning_conf/planning_config.pb.txt

```diff
@@ -1,24 +1,11 @@
-topic_config {
```

Comment:
need to change planning_config.pb.txt & planning.conf in branch 5.5 ，not master ;